x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils/wrap_search_source_fetch.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ISearchSource } from '@kbn/data-plugin/common';
+
+export function wrapSearchSourceFetch(abortController: AbortController) {
```

Comment:
Where this function is used? Can we document the purpose for its usage?

Reply:
This is no longer needed and will be removed. We decided to wrap entire search source instead of passing it through util fetch function.